editor/editor_log.cpp

```diff
@@ -220,6 +221,14 @@ void EditorLog::_meta_clicked(const String &p_meta) {
 			// Relative path. Convert to absolute, using executable path as reference.
 			path = path.trim_prefix("./").trim_prefix(".\\");
 			path = OS::get_singleton()->get_executable_path().get_base_dir().get_base_dir().path_join(path);
+		} else if (!FileAccess::exists(path)) {
+			// The file does not exist. Try on GitHub instead.
+			String branch = "master";
+			if (!str_compare(GODOT_VERSION_STATUS, "stable")) {
+				branch = vformat("%d.%d", GODOT_VERSION_MAJOR, GODOT_VERSION_MINOR);
+			}
```

Comment:
If we have the commit hash available, we can use it to open the exact source code on GitHub. This should be a perfect match, unless local uncommitted changes were performed before building.